### DISCUSSION TITLE
Error message following latest update 2.4.2

### DISCUSSION THREAD

**Discussion by Steve-M0SKM** [magicbug/Cloudlog#2128] [General]
On clicking the LOTW link I get the following message, the original config folder was left untouched during upgrade.

Any help gratefully received.

A Database Error Occurred

Error Number: 1054

Unknown column 'lotw_certs.cert_dxcc_id' in 'on clause'

SELECT `lotw_certs`.`lotw_cert_id` as `lotw_cert_id`, `lotw_certs`.`callsign` as `callsign`, `dxcc_entities`.`name` as `cert_dxcc`, `dxcc_entities`.`end` as `cert_dxcc_end`, `lotw_certs`.`qso_start_date` as `qso_start_date`, `lotw_certs`.`qso_end_date` as `qso_end_date`, `lotw_certs`.`date_created` as `date_created`, `lotw_certs`.`date_expires` as `date_expires`, `lotw_certs`.`last_upload` as `last_upload` FROM `lotw_certs` LEFT JOIN `dxcc_entities` ON `lotw_certs`.`cert_dxcc_id` = `dxcc_entities`.`adif` WHERE `user_id` = '6' ORDER BY `cert_dxcc` ASC

Filename: models/LotwCert.php

Line Number: 19

Steve M0SKM

**Comment by phl0** [comment]
Do you have access to the mySQL db? If so could you please check the value in migrations table?

**Reply by Steve-M0SKM** [reply]
Hello Florian, thank you for the reply.

Yes I have access to the db but it doesn't mean much to me. I see the following, where should I look?
Expand/Collapse
Structure
[migrations](https://phpmyadmin.eu.ionos.host/sql.php?server=1&db=dbs10652744&table=migrations&pos=0)
Expand/Collapse
[Columns](https://phpmyadmin.eu.ionos.host/tbl_structure.php?server=1&db=dbs10652744&table=migrations)
 Columns

    [New](https://phpmyadmin.eu.ionos.host/tbl_addfield.php?server=1&db=dbs10652744&table=migrations&field_where=last&after_field=)
    New
    Column
    [version](https://phpmyadmin.eu.ionos.host/tbl_structure.php?server=1&db=dbs10652744&table=migrations&field=version&change_column=1)

Steve

**Reply by phl0** [reply]
Are there supposed to be Screenshots? Sorry but those do not show up.

**Reply by phl0** [reply]
And please tell us the value of migrations in application/config/migrations.php line 24.

**Reply by Steve-M0SKM** [reply]
HI Florian,
Screen dump emailed to you + the text I found in line 24.
Thanks

**Comment by AndreasK79** [comment]
@Steve-M0SKM you said you left config-folder untouched? That means you didn't upgrade migration.php? Open that file in the config folder and tell us what number is here: $config['migration_version'] = 121; It should be 121. If not, that is the reason.

**Reply by Steve-M0SKM** [reply]
Hi Andreas,
Yes it is the original config folder.
I can't find $config['migration_version'] = 121 in the file migration.php

Steve

**Reply by phl0** [reply]
So which value does $config['migration_version'] have in your case?

**Reply by Steve-M0SKM** [reply]
Florian/Andreas
Thank you I have it working, I was looking at migration.php in the wrong folder. I have set it to 121 in the config folder and now Cloudlog is working.

Best 73
Steve

**Reply by phl0** [reply]
Ok, great. Can you please close this thread then?

**Comment by Steve-M0SKM** [comment]
Resolved thanks to Florian and Andreas.

**Reply by AndreasK79** [reply]
Happy you got it fixed!

**Reply by phl0** [reply]
Ur welcome. Glad its fixed.
